build-tools/packages/build-cli/src/commands/release/report.ts

```diff
@@ -567,11 +576,23 @@ export default class ReleaseReportCommand extends ReleaseReportBaseCommand<
 
 			const isNewRelease = this.isRecentReleaseByDate(latestDate);
 			const scheme = detectVersionScheme(latestVer);
-			const ranges = getRanges(latestVer);
+
+			if (context.flubConfig.releaseReport === undefined) {
+				throw new Error(`releaseReport not found in config.`);
+			}
 
 			// Expand the release group to its constituent packages.
 			if (isReleaseGroup(pkgName)) {
 				for (const pkg of context.packagesInReleaseGroup(pkgName)) {
+					assert(
+						pkg.monoRepo?.releaseGroup !== undefined,
+						`releaseGroup for ${pkgName} is undefined.`,
+					);
+					const ranges = getRanges(
+						latestVer,
+						context.flubConfig.releaseReport,
+						pkg.monoRepo?.releaseGroup,
+					);
```

Comment:
I find it unreliable to assert deep things `pkg.monoRepo?.releaseGroup`.
Here shouldn't this always be equal to pkgName (which is really pkgName or release group)?
You should be able to state `const releaseGroup: ReleaseGroup = pkgName;` just inside the if.
Unless `packagesInReleaseGroup` is not actually for release group, the should be able to `assert(pkg.monoRepo?.releaseGroup === releaseGroup`.

Reply:
pkgName is a release group. removing the assert.